common/sgx/eeid.c

```diff
@@ -438,15 +448,81 @@ static oe_result_t _verify_signature(
 
     result = OE_OK;
 
+done:
+    return result;
+}
+#else
+#ifdef _WIN32
+static oe_result_t _verify_signature(
```

Comment:
As a general design suggestion, instead of replicating `_verify_signature` 3x for different build configuration macros, consider defining a new `oe_rsa_public_key_from_modulus` method in openenclave/internal/rsa.h, and implementing that in the rsa.c for each of the supported crypto libs; verification should then have a common code path (load the key and verify signature through OE wrapper crypto functions) which reduces repetition of code and having to navigate the preprocessor options in this file. 

For reference, see the existing `oe_rsa_public_key_get_modulus/expoenent` methods, which are effectively symmetric with what you need here.

Reply:
Oh yes, you're absolutely right, thanks for the suggestion!

Reply:
(Done)